core/src/avm2/value.rs

```diff
@@ -57,6 +58,20 @@ static_assertions::assert_eq_size!(Value<'_>, [u8; 16]);
 #[cfg(target_pointer_width = "64")]
 static_assertions::assert_eq_size!(Value<'_>, [u8; 24]);
 
+impl fmt::Debug for Value<'_> {
```

Comment:
I don't think a manual impl is needed now - this can become a `#[derive(Debug)]` again

Reply:
A `#derive(Debug)]` would print the more verbose `Object(FooObject { ... })` instead of `FooObject`, but if you don't think this is worth it I can change it back

Reply:
I would prefer not to special-case the `Object` case like this (since the `Value::String` case gets printed with a wrapping `String`).